### ISSUE TITLE
Switch to conda?

### ISSUE THREAD

**Issue by kimt33** [theochem/horton#225]
I don't think the default linux package managers are reliable when installing dependencies for HORTON. There are too many errors that pop up every time the repo gets updated or if the repo is too old. It's quite difficult to keep the versions of each dependency to comply with HORTON. 
For example, Travis uses `apt-get` to install some outdated dependencies (#222), Fedora 25 (and onwards I think) uses Libxc 3.0.0 (#224), and Fedora automatically uses Atlas to build some of its .
Any chance we can just switch to a separate package manager? Like `pip install --user` fixes a lot of the python-related dependency problems. Explicitly downloading and building non-python packages from sources are kind of a pain, so maybe shifting to conda will be smart in the long run. 

I know @matt-chan has some sort of conda package/channel for HORTON stuff. We can build MKL, OpenBlas, Atlas all through conda so that would be nice too. It also has Mingw and some MSYS2 capabilities, so it's not too far fetched to get a conda package for windows as well. If we go for this modular structure subpackage structure, we can use conda to manage them too.

**Comment by tovrstra** [comment]
I agree that that would be very welcome in the 3.0 version. I would not present it as the only solution. It is always nice to be able to install HORTON on recent Linux distributions, without having to use an external package manager.

Package managers like Flatpak (http://flatpak.org/) and Snap (https://snapcraft.io/) could also be interesting. These should be reasonably well supported in recent Linux distributions.
